includes/definitions/discovery/uhp.yaml

```diff
@@ -1,9 +1,9 @@
 mib: UHP-MIB
 modules:
   os:
-    hardware: ENTITY-MIB::entPhysicalDescr.1 
-    version: swVersion
-    serial: serialNumber   
+    hardware: RFC1213-MIB::sysDescr.0 
```

Comment:
sysDescr is already fetched, use sysDescr_regex to paste the hardware out of it.

Reply:
Should be good now.